.circleci/config.yml

```diff
@@ -261,11 +263,14 @@ workflows:
          cache-key: v29-packaging-dependencies-{{ checksum "client/package-lock.json"}}
          executor: with-chrome
          browser: chrome
+         env: 'service-key=${FIRE}'
          requires:
             - install-client-dependencies  
          install-command: |
             npm install -f 
             npm install apollo-angular @apollo/client graphql
+            npm i cypress-firebase
+            npm i firebase-admin
          pre-steps: 
             - run: npm install -g concurrently
```

Comment:
Pourquoi tu installe toutes les dépendances à la main comme cela ?
C'est le principe des devDependancy pour y mettre les dépendances de dev.

Reply:
oui je vais les enlever, c'était pour être sûr qu'il y ait , parce-que je suppose puisque j'ai mis les dependencies à save-cache du coup la deuxième fois quand il cherche et il les trouve dans CircleCI, il re-install pas toutes les dependencies. pour cela je dois changer la clé cache à chaque ajournement d'une dependencie.

Reply:
https://circleci.com/docs/2.0/caching/#using-keys-and-templates

Reply:
ah je savais pas que checksum vérifie les changements, merci!